api/Selection.json

```diff
@@ -0,0 +1,317 @@
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
+            "deprecated": false
+          }
+        }
+      },
+      "setBaseAndExtent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Selection/setBaseAndExtent",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "53"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": "53"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "deleteFromDocument": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Selection/deleteFromDocument",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "55"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": "55"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "empty": {
+        "__compat": {
+          "description": "<code>empty()</code> as alias of <code>removeAllRanges()</code>",
```

Comment:
I think it makes sense to include the MDN URL https://developer.mozilla.org/docs/Web/API/Selection/removeAllRanges